controllers/drplacementcontrol_controller_test.go

```diff
@@ -586,7 +586,8 @@ func createDRPolicy(inDRPolicy *rmn.DRPolicy) {
 }
 
 func createDRPolicyAsync() {
-	createDRPolicy(asyncDRPolicy)
+	policy := asyncDRPolicy.DeepCopy()
```

Comment:
Does Create update the object it created if succesful? I  was under the assumption that if we use Get then turnaround and use the object it may complain about unwanted fields, was not aware Created resources have a similar problem.

IOW, does create need a deep copy before creation?

Reply:
Yes. It is correct. i.e. it happens with create as well similar to get.